service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/database/CloneControlledAzureDatabaseResourceFlightTest.java

```diff
@@ -0,0 +1,168 @@
+import bio.terra.workspace.service.job.JobMapKeys;
+import bio.terra.workspace.service.job.JobService;
+import bio.terra.workspace.service.resource.controlled.cloud.azure.database.AzureDatabaseUtilsRunner;
+import bio.terra.workspace.service.resource.controlled.flight.clone.azure.common.ClonedAzureResource;
+import bio.terra.workspace.service.resource.model.CloningInstructions;
+import bio.terra.workspace.service.resource.model.WsmResourceType;
+import bio.terra.workspace.service.workspace.WorkspaceService;
+import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys;
+import bio.terra.workspace.service.workspace.model.Workspace;
+import java.time.Duration;
+import java.util.UUID;
+import org.junit.jupiter.api.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Tag("azureConnected")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CloneControlledAzureDatabaseResourceFlightTest extends BaseAzureConnectedTest {
+
+  private static final Logger logger =
+      LoggerFactory.getLogger(CloneControlledAzureDatabaseResourceFlightTest.class);
+
+  @Autowired private JobService jobService;
+  @Autowired private AzureTestUtils azureTestUtils;
+  @Autowired private WorkspaceService workspaceService;
+  @Autowired private UserAccessUtils userAccessUtils;
+  @Autowired private AzureDatabaseUtilsRunner azureDatabaseUtilsRunner;
+
+  private Workspace sourceWorkspace;
+  private Workspace destinationWorkspace;
+
+  @BeforeAll
+  public void setup() throws InterruptedException {
+    sourceWorkspace =
+        createWorkspaceWithCloudContext(workspaceService, userAccessUtils.defaultUserAuthRequest());
+    destinationWorkspace =
+        createWorkspaceWithCloudContext(workspaceService, userAccessUtils.defaultUserAuthRequest());
+  }
+
+  @AfterAll
+  public void cleanup() {
+    // Deleting the workspace will also delete any resources contained in the workspace, including
+    // VMs and the resources created during setup.
+    workspaceService.deleteWorkspace(sourceWorkspace, userAccessUtils.defaultUserAuthRequest());
+    workspaceService.deleteWorkspace(
+        destinationWorkspace, userAccessUtils.defaultUserAuthRequest());
+  }
+
+  @Test
+  void cloneControlledAzureDatabaseFlightTest() throws InterruptedException {
+    FlightMap inputs = new FlightMap();
+
+    // To fill in this value:
+    // Go to your workspace dashboard -> cloud information -> copy the "Storage SAS URL"
+    inputs.put("BLOB_CONTAINER_URL_AUTHENTICATED", "https://...");
+
+    // To fill in this value:
+    // Go to your workspace dashboard -> cloud information -> copy the value starting with "sc-"
+    //   from either the "Storage SAS URL" or the "Storage Container URL".
+    // Or: use "sc-${WORKSPACE_ID}" (our conventional format for storage containers names)
+    inputs.put(
+        WorkspaceFlightMapKeys.ControlledResourceKeys.AZURE_STORAGE_CONTAINER_NAME, "sc-...");
```

Comment:
You'll also need to fill in this value until the flight is updated to include storage container creation and deletion (See [WM-2366](https://broadworkbench.atlassian.net/browse/WM-2366)